src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualHub.md

```diff
@@ -113,3 +113,5 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## NOTES
 
 ## RELATED LINKS
```

Comment:
Please remove the second ## related links from these files - not sure how that got there.